src/physics/PhysicsSystem.java

```diff
@@ -84,24 +84,30 @@ public void update() {
 				currTrans.increasePosition(newVeloc.mul(super.getDeltaTime()));
 			}
 		}
-		
+
 		super.timeMarkEnd();
 	}
-	
+
 	public void cleanUp() {
-		
+
 	}
-	
+
 	public void loadBlueprint(ArrayList<String> blueprint) {
 		// - PhysicsComponent
 		ArrayList<String> physicsComponentData = BlueprintLoader.getAllLinesWith("PHYSICSCOMPONENT", blueprint);
 		String[] frags = null;
-		for(String dataSet : physicsComponentData) {
-			int eID = BlueprintLoader.extractEID(dataSet);
-			frags = BlueprintLoader.getDataFragments(dataSet);
-			entityController.addPhysicsComponent(eID)
-				.setWeight(Float.valueOf(frags[0]))
-				.setAffectedByGravity(Boolean.valueOf(frags[1]));
+		for (String dataSet : physicsComponentData) {
+			try {
+				int eID = BlueprintLoader.extractEID(dataSet);
+				if (eID < 0) {
+					continue;
```

Comment:
IMO Malformed input lines shouldn't just be discarded. Makes it a pain to debug why some behavior isn't happening even though you think you defined a component for it.

Reply:
is a warning in sys.err sufficient? or do you expect an exception?

Reply:
A syserr warning should be fine